### ISSUE TITLE
tap-facebook does not currently support Account-level Insights Reports

### ISSUE THREAD

**Issue by dylan-stitch** [singer-io/tap-facebook#46]
The tap makes requests at the ad-level, which to my knowledge is not configurable, though [this block](https://github.com/singer-io/tap-facebook/blob/master/tap_facebook/__init__.py#L390) suggests that it could be.

Pulling insights reports for an Account would provide higher-level information about account performance.

**Comment by bradcosine** [comment]
@dylan-stitch I agree... in my case I want to set level='adset' and ultimately had to edit the __init__.py file directly as I could not see a simple way to configure it
